src/redux/actions/childAction.js

```diff
@@ -0,0 +1,43 @@
+import axios from 'axios';
+export const ERROR_ACTION = 'ERROR_ACTION';
+export const GET_CHILDREN_ACTION = 'GET_CHILDREN_ACTION';
+export const GET_CHILDREN_SUCCESS = 'GET_CHILDREN_SUCCESS';
+export const GET_COURSES_ACTION = 'GET_COURSES_ACTION';
+export const GET_COURSES_SUCCESS = 'GET_COURSES_SUCCESS';
+
+const token = localStorage.getItem('okta-token-storage');
+const parsedToken = JSON.parse(token);
+
+const axiosWithAuth = () => {
```

Comment:
should be importing the built out util file axiosWithAuth instead of declaring your own